[CTOR-977] Refresh network-efficientip-snmp doc page (#3668)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-efficientip-snmp.md

| CRITICALXFRREJ          | Critical threshold.  General name server statistics: 'udp', 'tcp', 'requestv4', 'requestv6', 'recursion', 'response', 'recurserej', 'duplicate', 'dropped', 'res-queryv4', 'res-queryv6', 'res-retry', 'res-responsev4', 'res-responsev6'. DNS answers statistics: 'success', 'formerr', 'servfail', 'nxdomain', 'nxrrset', 'failure'. DNS Transfer Requests Statistics: 'xfrdone', 'xfrrej'. DNSSEC Validation Statistics: 'res-val', 'res-valsuccess', 'res-valnegsuccess', 'res-valfail' |                                                                                                                                                              |             |

> | EXTRAOPTIONS            | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                              |             |

Reviewer: ```suggestion
| EXTRAOPTIONS            | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                              |             |
```

| CRITICALXFRREJ          | Critical threshold.  General name server statistics: 'udp', 'tcp', 'requestv4', 'requestv6', 'recursion', 'response', 'recurserej', 'duplicate', 'dropped', 'res-queryv4', 'res-queryv6', 'res-retry', 'res-responsev4', 'res-responsev6'. DNS answers statistics: 'success', 'formerr', 'servfail', 'nxdomain', 'nxrrset', 'failure'. DNS Transfer Requests Statistics: 'xfrdone', 'xfrrej'. DNSSEC Validation Statistics: 'res-val', 'res-valsuccess', 'res-valnegsuccess', 'res-valfail' |                     |             |
| EXTRAOPTIONS            | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                                                                                                                                                                                                                                                                                                                          |                     |             |

</TabItem>
<TabItem value="Dns-Usage" label="Dns-Usage">

> | Macro                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Valeur par défaut | Obligatoire |

Reviewer: ```suggestion
```
